diff -up pgbadger-13.0/pgbadger.orig pgbadger-13.0/pgbadger
--- pgbadger-13.0/pgbadger.orig	2025-02-06 15:34:37.424769979 +0100
+++ pgbadger-13.0/pgbadger	2025-02-06 15:34:47.988986824 +0100
@@ -43,7 +43,7 @@ use Time::Local qw(timegm_nocheck timelo
 use POSIX qw(locale_h sys_wait_h _exit strftime);
 setlocale(LC_NUMERIC, '');
 setlocale(LC_ALL,     'C');
-use File::Spec qw/ tmpdir /;
+use File::Spec;
 use File::Temp qw/ tempfile /;
 use IO::Handle;
 use IO::Pipe;
